### ISSUE TITLE
Sending ether from BSCScan contract wallet to CoinDCX exchange failed..can anyone help...

### ISSUE THREAD

**Issue by MJDhakra** [ethereum/meteor-dapp-wallet#512]
![BSCScan](https://user-images.githubusercontent.com/78607806/107054306-f5989c80-67f5-11eb-9ab9-b7ac418fef1e.jpg)
Hi,
I transferred ether from binance using bscscan to coindcx wallet exchange. However they have not received the ether in their account yet and it has ben 12 days. 

Transaction # is 0x31b356a063e868e15d74c1bab6c96e9b98499dbb500fad9f52b3c182f481515c

Transferred from - 0x631fc1ea2270e98fbd9d92658ece0f5a269aa161
Transferred to - https://bscscan.com/token/0x2170ed0880ac9a755fd29b2688956bd959f933f8?a=0x6daf077f8eea2b1572ac6d2f73a58066586b26c4

The transaction status shows as success on bscscan but when i contacted the exchange they said they haven't received the coin. 

I have attached a screen grab of the transaction from bscscan. Any help would be really appreciated.

**Comment by airboat** [comment]
Yes and keep trying get to it but each time password  issues and I use the same it work one and won't the next trying to move to meta mask or coin base having issues
